pipeline/stop_places/sql/stop_places.sql

```diff
@@ -532,7 +532,7 @@ CREATE OR REPLACE VIEW final_access_spaces AS (
   SELECT ptr.relation_id, acc.*
   FROM access_spaces acc
   JOIN stop_areas_members_ref ptr
-    ON acc.osm_id = ptr.member_id AND acc.osm_type = ptr.osm_type
```

Comment:
This should contain `AND ptr.osm_type = 'N'` like it is done for the final_entrances view.